pkg/datapath/linux/node_addressing.go

```diff
@@ -67,7 +64,32 @@ func listLocalAddresses(family int) ([]net.IP, error) {
 	return addresses, nil
 }
 
-type addressFamilyIPv4 struct{}
+func listSubnetAddresses(nets []net.IPNet) []net.IP {
+	var ips []net.IP
+	for _, nt := range nets {
+		for ip := nt.IP.Mask(nt.Mask); nt.Contains(ip); ip = incIP(ip) {
+			ips = append(ips, ip)
+		}
+	}
```

Comment:
Correct me if I'm wrong, won't Cilium memory spike by ~64MB during initialization the because of this for loop if the user sets `--node-port-addresses=127.0.0.0/8`? Can't we using the CIDRs all over the place instead?